apps/etterlatte-saksbehandling-ui/client/src/components/aktivitetsplikt/stegmeny/AktivitetspliktStegmeny.tsx

```diff
@@ -0,0 +1,22 @@
+import { StegMenyBox } from '~components/behandling/StegMeny/stegmeny'
+import { HStack } from '@navikt/ds-react'
+import React from 'react'
+import { AktivitetNavLenke } from '~components/aktivitetsplikt/stegmeny/AktivitetspliktNavLenke'
+
+export enum AktivitetspliktSteg {
+  VURDERING = 'vurdering',
+  BREV = 'brev',
+}
+
+export function AktivitetspliktStegmeny() {
+  const skalBrevVises = true // TODO: logikk for om man skal sende brev
```

Comment:
sendes og vises eller en av de to?

Reply:
Hehe, litt uavklart. Det kommer nok litt på det senere